osu.Game.Tests/Visual/Multiplayer/TestSceneTeamVersus.cs

```diff
@@ -68,6 +69,8 @@ public override void SetUpSteps()
                 LoadScreen(dependenciesScreen = new DependenciesScreen(client));
             });
 
+            AddUntilStep("wait for dependencies screen", () => Stack.CurrentScreen is DependenciesScreen);
+            AddUntilStep("wait for dependencies to start load", () => dependenciesScreen.LoadState > LoadState.Ready);
```

Comment:
Shouldn't this be

```suggestion
            AddUntilStep("wait for dependencies to start load", () => dependenciesScreen.LoadState > LoadState.NotLoaded);
```

The enum goes `NotLoaded` -> `Loading` -> `Ready` -> `Loaded`, so wouldn't `> Ready` just amount to another `IsLoaded` check...?